data/lib/src/network/linshare_http_client.dart

```diff
@@ -67,9 +69,18 @@ class LinShareHttpClient {
   }
 
   Future<bool> deletePermanentToken(PermanentToken token) async {
-    final deletedToken = await _dioClient.delete(
-      Endpoint.authentication.generateEndpointPathWithParameter(token.tokenId.uuid)
-    );
+    // final headerParam = _dioClient.getHeaders();
+    // headerParam.clear();
+    // headerParam[HttpHeaders.acceptHeader] = 'application/json';
+    // headerParam[HttpHeaders.contentTypeHeader] = 'application/json';
+
+    final deletedToken = await http.delete('https://user.linshare-2-3.integration-linshare.org/' + Endpoint.authentication.generateEndpointPathWithParameter(token.tokenId.uuid));
```

Comment:
This throws 401 (Unauthorized)

Reply:
Why you need to update the `path`? Previous solution is ok for me